build/Makefile

```diff
@@ -433,29 +435,41 @@ ensure-windows-buildx:
 	#
 	# Windows image builds must be directed to a specific buildx context.
 	# The default context does not support building cross platform images.
-	-DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name=$(BUILDX_WINDOWS_BUILDER) --use
+	-docker buildx create --name=$(BUILDX_WINDOWS_BUILDER) 
 
 # Build the image for the gameserver sidecar and SDK binaries
-build-agones-sdk-image: $(ensure-build-image) build-agones-sdk-binary build-licenses build-required-src-dist
+build-agones-sdk-image: $(ensure-build-image) build-licenses build-required-src-dist build-agones-sdk-image-amd64 
+ifeq ($(WITH_ARM64), 1)
+build-agones-sdk-image: $(ensure-build-image) $(ensure-arm-builder) build-licenses build-required-src-dist build-agones-sdk-image-arm64
```

Comment:
I don't think this change is actually necessary - I say that for two reasons:
1. `ensure-build-image` already has `ensure-arm-builder` as a dependency, so it will run regardless if needed.
2. There is no `ensure-arm-builder` variable, so I don't think it's doing anything anyway.